tests/functional/spec/features/grouped_validation/sum/grouped_validation_sum.spec.js

```diff
@@ -0,0 +1,299 @@
+const helpers = require('../../../../helpers');
+
+describe('Feature: Sum of grouped answers validation against total', function() {
+  describe('Equal to', function () {
+    describe('Given I start a grouped answer validation survey and enter 12 into the total', function() {
+
+      var TotalAnswerPage = require('../../../../pages/features/grouped_validation/sum/equal-to/total-block.page');
+      var BreakdownAnswerPage = require('../../../../pages/features/grouped_validation/sum/equal-to/breakdown-block.page');
+      var SummaryPage = require('../../../../pages/features/grouped_validation/sum/equal-to/summary.page');
+
+      before(function() {
+        return helpers.openQuestionnaire('test_sum_equal_validation_against_total.json');
+      });
+
+      it('When I continue and enter 3 in each breakdown field, Then I should be able to get to the summary', function() {
+        return browser
+         .setValue(TotalAnswerPage.answer(), '12')
+         .click(TotalAnswerPage.submit())
+         .setValue(BreakdownAnswerPage.breakdown1(), '3')
+         .setValue(BreakdownAnswerPage.breakdown2(), '3')
+         .setValue(BreakdownAnswerPage.breakdown3(), '3')
+         .setValue(BreakdownAnswerPage.breakdown4(), '3')
+         .click(BreakdownAnswerPage.submit())
+         .getUrl().should.eventually.contain(SummaryPage.pageName);
+        });
+     });
+  });
+
+   describe('Equal to', function () {
+    describe('Given I start a grouped answer validation survey and enter 5 into the total', function() {
+
+      var TotalAnswerPage = require('../../../../pages/features/grouped_validation/sum/equal-to/total-block.page');
+      var BreakdownAnswerPage = require('../../../../pages/features/grouped_validation/sum/equal-to/breakdown-block.page');
+      var SummaryPage = require('../../../../pages/features/grouped_validation/sum/equal-to/summary.page');
+
+      before(function() {
```

Comment:
There is a lot of duplication here. The first `describe('Equal to', function () {` on line 4 should wrap all the others and that is where you declare the vars and before function. Then further down this is the same for less than etc.

Reply:
Not done.

Reply:
Still lots of duplication, the before functions should be shared

Reply:
@shahi645 You've not removed the duplication of `before` or `var` declaration. That can all be done within the initial `describe('Equal to', function () {` which then wraps the preceding describes.

Reply:
surely now i get a tick